definitions/checks/foreman_proxy/check_tftp_storage.rb

```diff
@@ -0,0 +1,43 @@
+module Checks::ForemanProxy
+  class CheckTftpStorage < ForemanMaintain::Check
+    metadata do
+      label :check_tftp_storage
+      description 'Clean old Kernel and initramfs files from tftp-boot'
+      tags :default
+      confine do
+        feature(:satellite)
+      end
+    end
+
+    def run
+      tftp_boot_dir = feature(:foreman_proxy).tftp_root_directory + '/boot/'
+      if !feature(:foreman_proxy).features.include?('tftp') || token_duration == 0
+        skip 'Skipping the check as TFTP feature is disabled or token duration is zero'
+      elsif Dir.exist?(tftp_boot_dir)
+        files = files_to_delete(tftp_boot_dir)
+        assert(files.empty?,
+               'There are old initrd and vmlinuz files present in tftp',
+               :next_steps => Procedures::Files::Remove.new(:files => files))
+      else
+        warn! "TFTP root directory #{tftp_boot_dir} does not exist."
```

Comment:
Can't we skip this check when directory doesn't exist?
OR if we really want to show this message then I will prefer `skip` over `warn!`.

Reply:
OK, I will replace it with `skip`.